cubecrit/templates/manufacturer.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% block title %}
+    {{ manufacturer.display_name }}
+{% endblock title %}
+{% block content %}
+    <h1 class="display-2 text-center">{{ manufacturer.display_name }}</h1>
+    <blockquote class="blockquote text-bg-light p-3 w-75 m-auto mb-3">
+        <p>{{ manufacturer.bio }}</p>
+    </blockquote>
+    <hr>
+    <div class="row justify-content-center">
+        {% for row in puzzle_page %}
+            <div class="col-lg-3 col-6">
+                <div class="card">
+                    <img src="{{ url_for('static', filename='/assets/cube.png') }}"
+                         class="card-img-top"
+                         style="width: 100px"
+                         alt="...">
+                    <div class="card-body">
+                        <h5 class="card-title">{{ row.manufacturer.display_name }} {{ row.display_name }}</h5>
+                        <p class="card-text"></p>
```

Comment:
Remove this empty tag if we aren't putting any content in it (we can add it back later if we do have content to put here)